### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by lukechampine**

### CODE REVIEW

File: app/plugins/Wallet/js/buttons.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function filterAddressList(searchstr) {
 	NodeList.prototype.forEach = Array.prototype.forEach
 	var entries = eID('address-list').childNodes;
 	entries.forEach( function(entry) {
-		if (entry.querySelector('.address').innerHTML.indexOf(searchstr) || searchstr.length == 0) {
+		if (entry.querySelector('.address').innerHTML.indexOf(searchstr)) {
```
Comment (DavidVorick) [inline app/plugins/Wallet/js/buttons.js:26]: one of these things is not like the others

File: release.sh [hunk 2]
```diff
@@ -5,9 +5,9 @@
 # constants
 readonly RELEASE_DIR="release"
 readonly ELECTRON_URL="https://github.com/atom/electron/releases/download"
-readonly ELECTRON_VERSION="v0.33.4"
+readonly ELECTRON_VERSION="v0.34.0"
```
Comment (DavidVorick) [inline release.sh]: prefer not to use a `.0` release, would rather stick to `33.9`

Comment (lukechampine) [inline release.sh]: I think their `.0` releases are a lot more conservative than ours, lol

Comment (DavidVorick) [inline release.sh]: :|

Comment (DavidVorick) [inline release.sh]: actually, looking at the changelogs, seems like they make fairly arbitrary decisions about when to bump the major number

**Pull request merged**